addons/skin.confluence/720p/MyPVRTimers.xml

```diff
@@ -160,6 +160,14 @@
 						</control>
 						<control type="group">
 							<visible>!IsEmpty(ListItem.Date)</visible>
+							<control type="image">
+								<left>0</left>
+								<top>8</top>
+								<width>50</width>
+								<height>26</height>
+								<texture border="1">$INFO[ListItem.Icon]</texture>
```

Comment:
Don't know. As I said I simply restored the previous code. I'm far away from being a skinner. 
If the border attribute is not needed, let's remove it in another PR.